fieldsolver/ldz_main.cpp

```diff
@@ -53,6 +53,8 @@
 #include "fs_limiters.h"
 #include "mpiconversion.h"
 #include "../fieldtracing/fieldtracing.h"
+#include "../logger.h"
+extern Logger logFile;
```

Comment:
This needs logfile and didn't have it before?

Reply:
For some reason, CI was complaining about this back in last April, so I threw it in.